src/adapters/bidderFactory.js

```diff
@@ -107,6 +107,8 @@ import { logWarn, logError, parseQueryStringParameters, delayExecution } from 's
  * @property {string} url The URL which makes the sync happen.
  */
 
+const BID_RESPONSE_KEYS = ['currency', 'netRevenue', 'ttl'];
```

Comment:
We have more required params. You should check http://prebid.org/dev-docs/bidder-adapter-1.html#interpreting-the-response ;)

Reply:
Updated